Sources/BraintreePayPal/BTPayPalAccountNonce.m

```diff
@@ -16,6 +18,75 @@ @interface BTPayPalAccountNonce ()
 
 @implementation BTPayPalAccountNonce
 
+- (nullable instancetype)initWithJSON:(BTJSON *)json {
+    BTJSON *payPalAccount = json[@"paypalAccounts"].asArray.firstObject;
+    NSString *nonce = payPalAccount[@"nonce"].asString;
+    BOOL isDefault = payPalAccount[@"default"].isTrue;
+
+    if (self = [super initWithNonce:nonce type:@"PayPal" isDefault:isDefault]) {
+        BTJSON *details = payPalAccount[@"details"];
+        _email = details[@"email"].asString ?: details[@"payerInfo"][@"email"].asString;
+        _clientMetadataID = details[@"correlationId"].asString;
```

Comment:
Is the correlationId the same as clientMetadataID? If so, should the clientMetadataID be passed for correlationID in the TODO below?

Reply:
Yes, it should be the same value. But we need to get it from the Native Checkout SDK after the customer completes the UI flow and pass it to the Braintree Gateway when we tokenize. Here, we're parsing it from the gateway tokenization response.